lib/onceler/recorder.rb

```diff
@@ -92,7 +81,7 @@ def record!
 
     def reset!
       run_before_hooks(:reset)
-      rollback_transactions!
+      @tape.teardown_fixtures
```

Comment:
what's our fallback in case this instance variable doesn't for some reason get initialized